_posts/2021-01-07-Work-Day-Progress.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: post
+title: Work Day Progress
+description: 
+summary: 
+tags:
+---
+ Today I ended up finishing my research and I started again with the coding on Sublime text. We started off the day with Typing Club for a bit and then I looked through Sublime text and what code scripts I needed to add pictures onto my Website. It was really interesing to learn more about our planet and what is going on around us. At the start of this project, I did think that Climate change was a problem, but now I really think that it should be someting that we are focusing on and we should be taking steps to save our planet. 
```

Comment:
`finishing my research and I` <-- it's a bit vague as we don't know what research you are talking about. Research on what?

Reply:
You end with what you thought about climate change. But what does that have to do with your website? What change did you make to your website?